.github/workflows/codeql-analysis-windows.yml

```diff
@@ -0,0 +1,62 @@
+name: "CodeQL for Windows"
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ master ]
+  schedule:
+    - cron: '23 11 * * 6'
+
+jobs:
+  analyze-windows:
+    name: Analyze Windows (Server 2019)
+    runs-on: windows-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        language: [ 'cpp' ]
+        #language: [ 'cpp', 'javascript', 'python' ]
+        # CodeQL supports [ 'cpp', 'csharp', 'go', 'java', 'javascript', 'python' ]
+        # Learn more:
+        # https://docs.github.com/en/free-pro-team@latest/github/finding-security-vulnerabilities-and-errors-in-your-code/configuring-code-scanning#changing-the-languages-that-are-analyzed
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
+    - name: Install Qt
+      uses: jurplel/install-qt-action@v2
+      with:
+        version: '5.15.2'
+        dir: '${{ github.workspace }}'
+        arch: 'win64_msvc2019_64'
```

Comment:
Do we only need 64 bit?

Reply:
For the CodeQL?  I don't really know -- I'm assuming it's not _that_ dependent on the OS architecture.

Otherwise we would need to do each OS, on each ARCH, plus perhaps headless and nosound combinations.  (I run my server headless+nosound, so if that affects the CodeQL, I'd want it covered.)

Reply:
I don't know either. So probably that's ok.

Reply:
In fact, from the results, I'm wondering if it needs to be run multi-platform even.

Reply:
If you run it multi-platform, issues in the sound.cpp and sound.h files can be checked for every OS.

Reply:
Possibly - I don't know how it chooses which files to scan.  If it's based on the object files created by the build, then that would do it.